data/transactions/logic/crypto_test.go

```diff
@@ -423,7 +423,6 @@ byte 0x%s
 		{pkTampered2, false},
 	}
 	for i, test := range decompressTests {
-		i, test, source := i, test, source
```

Comment:
`src := fmt.Sprintf(source` below now modifies the original `source` program

Reply:
Think this is addressed in the latest push.